docs/source/advanced/mixed_precision.rst

```diff
@@ -5,49 +5,154 @@
+#########
+Precision
+#########
+
+There are numerous benefits to using numerical formats with lower precision than the 32-bit floating-point or higher precision such as 64-bit floating-point.
+
+Lower precision, such as the 16-bit floating-point, enables the training and deployment of large neural networks since they require less memory, enhances data transfer operations since they require
+less memory bandwidth and run batch operations much faster on GPUs that support Tensor Core. [`1 <https://docs.nvidia.com/deeplearning/performance/mixed-precision-training/index.html>`_].
+
+Higher precision, such as the 64-bit floating-point, can be used for highly sensitive use-cases.
+
+Following are the precisions available in Lightning along with their supported Accelerator:
+
+.. list-table:: Precision with Accelerators
+   :widths: 20 20 20 20 20
+   :header-rows: 1
+
+   * - Precision
+     - CPU
+     - GPU
+     - TPU
+     - IPU
+   * - 16
+     - No
+     - Yes
+     - No
+     - Yes
+   * - BFloat16
+     - Yes
+     - Yes
+     - Yes
+     - No
+   * - 32
+     - Yes
+     - Yes
+     - Yes
+     - Yes
+   * - 64
+     - Yes
+     - Yes
+     - No
+     - No
+
+
+***************
+Mixed Precision
+***************
+
+PyTorch, like most deep learning frameworks, trains on 32-bit floating-point (FP32) arithmetic by default. However, many deep learning models do not require this to reach complete accuracy. By conducting
+operations in half-precision format while keeping minimum information in single-precision to maintain as much information as possible in crucial areas of the network, mixed precision training delivers
+significant computational speedup. Switching to mixed precision has resulted in considerable training speedups since the introduction of Tensor Cores in the Volta and Turing architectures. It combines
+FP32 and lower-bit floating-points (such as FP16) to reduce memory footprint and increase performance during model training and evaluation. It accomplishes this by recognizing the steps that require
+complete accuracy and employing a 32-bit floating-point for those steps only, while using a 16-bit floating-point for the rest. When compared to complete precision training, mixed precision training
+delivers all of these benefits while ensuring that no task-specific accuracy is lost. [`2 <https://docs.nvidia.com/deeplearning/performance/mixed-precision-training/index.html>`_].
 
 .. note::
 
-    In some cases it is important to remain in FP32 for numerical stability, so keep this in mind when using mixed precision.
+    In some cases, it is essential to remain in FP32 for numerical stability, so keep this in mind when using mixed precision.
+    For example, when running scatter operations during the forward (such as torchpoint3d), computation must remain in FP32.
+
+.. warning::
+
+    Do not cast anything to other dtypes manually using ``torch.autocast`` when using native precision because
```

Comment:
```suggestion
    Do not cast anything to other dtypes manually when using native precision because
```

This is not only true for `torch.autocast` but also if people call `.half` or something similar

Reply:
updated!